node/modules/actorevent.go

```diff
@@ -63,9 +65,20 @@ func EthEventAPI(cfg config.ActorEventConfig) func(helpers.MetricsCtx, fx.Lifecy
 
 		// Enable indexing of actor events
 		var eventIndex *filter.EventIndex
-		if cfg.EnableHistoricFilterAPI {
+		if !cfg.DisableHistoricFilterAPI {
+			var dbPath string
+			if cfg.ActorEventDatabasePath == "" {
```

Comment:
I wonder if we can simplify this to DatabasePath, or do we need to qualify?

Reply:
Yep, we can do that!